src/depackers/uncompress.c

```diff
@@ -90,7 +90,7 @@ static int decrunch_compress(FILE * in, FILE * out, long inlen)
 	block_mode = inbuf[2] & BLOCK_MODE;
 	maxmaxcode = MAXCODE(maxbits);
 
-	if (maxbits > BITS) {
+	if (maxbits < INIT_BITS || maxbits > BITS) {
 		/*fprintf(stderr,
```

Comment:
Is this, or an equivalent of it, applied in any Linux distros? Can't find it in https://github.com/vapier/liblzw/blob/main/lzw.c

Reply:
I don't know, I haven't checked. A maxbits value under 9 is nonsensical because the initial bits for this type of LZW is always 9. [POSIX claims a valid under 9 for the maxbits argument](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/compress.html) to `compress` is invalid.

Reply:
OK.